### ISSUE TITLE
Linking with password fails

### ISSUE THREAD

**Issue by joost** [tutumcloud/mongodb#30]
I'm trying to link together a meteor app with the mongodb container. However it seems that the meteor app fails very quickly since the mongodb-user is not created fast enough. Not sure.. but can this be the case?

**Comment by Levino** [comment]
No, please post the logs from the Meteor container. If meanwhile fixed this should be closed.
